content/logs/faq/why-my-logs-do-not-have-the-expected-timestamp.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Why my logs do not have the expected timestamp?
```

Comment:
I would change this to ` Why do my logs not have the expected timestamp?`

Reply:
👍